src/lib/prov/openssl/openssl_rc4.cpp

```diff
@@ -45,6 +45,11 @@ class OpenSSL_RC4 : public StreamCipher
 
       explicit OpenSSL_RC4(size_t skip = 0) : m_skip(skip) { clear(); }
       ~OpenSSL_RC4() { clear(); }
+
+      void seek(u64bit offset) override
+      {
+        (void)offset;
```

Comment:
I understand why you did this, but this should throw an Exception instead. In the case of RC4, calling `seek` is an application logic error, so this particular case should be `throw Exception("RC4 does not support seeking")`